packages/client/tests/functional/issues/22098-column_does_not_exist/test.ts

```diff
@@ -0,0 +1,12 @@
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { PrismaClient } from './node_modules/@prisma/client'
+
+declare let prisma: PrismaClient
+
+testMatrix.setupTestSuite(() => {
+  test('does not throw error', () => {
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises, jest/valid-expect
+    expect(prisma.test.findFirst()).resolves.not.toThrow()
```

Comment:
Went with the following solution, unfortunately, eslint doesn't seem to be able to pick up what's happening with the promise